scripts/mppc_to_multiprobe.py

```diff
@@ -0,0 +1,242 @@
+image. This corresponds to the offset, which can be corrected by adjusting the position of the galvo mirrors.
+"""
+from __future__ import division
+
+import argparse
+import logging
+import math
+import numpy
+
+from odemis import model
+from odemis.dataio import hdf5
+from odemis.util import almost_equal
+from scipy.signal import correlate
+import sys
+from builtins import input
+
+from odemis.acq.align.spot import FindSpot
+
+
+def get_mppc_rotation(image):
+    """
+    Calculates the rotation of the MPPC detector in space. MPPC detector should be horizontally positioned.
+    The scanning is performed with one beamlet over a field of view larger than the effective size of the single
+    MPPC cells.
+    TODO For now all cells are active. => Need to cut the respective mppc cell images.
+    TODO: specify if all 4 mppc cells can be activated during the same scan or if 4 images are needed.
+        If possible to activate all 4 cells in same scan, need multi spot detection.
+    Note: Image needs to be acquired on homogeneous non-sample area.
+    :param image: (ndarray) The full mppc detector image to run the calibration for.
+    :returns: (tuple of 2 float) The difference of the corner cells A8 to A1 in x and the difference
+                of H1 to A1 in y.
+    """
+    # Get the 4 corner mppc cell images: A1, H1, A8, H8
+    # Size of one mppc cell image is (800, 800) TODO @Job correct?
+    # Shape of full mppc is (8, 8)
+    image_A1 = image[0:800, 0:800]
+    image_H1 = image[-801:-1, 0:800]
+    image_A8 = image[0:800, -801:-1]
+    image_H8 = image[-801:-1, -801:-1]
+
+    # find the center of all 4 mppc cell images
+    mppc_cell_spotcenter_A1 = FindSpot(image_A1)  # tuple [px]
+    mppc_cell_spotcenter_A8 = FindSpot(image_A8)  # tuple [px]
+    mppc_cell_spotcenter_H1 = FindSpot(image_H1)  # tuple [px]
+    mppc_cell_spotcenter_H8 = FindSpot(image_H8)  # tuple [px]
+
+    # compare A1 with H1: should have same y
+    delta_y1 = mppc_cell_spotcenter_A1[1] - mppc_cell_spotcenter_H1[1]  # pos: CCW rotation, neg: CW rotation
+    # compare A8 with H8: should have same y
+    delta_y2 = mppc_cell_spotcenter_A8[1] - mppc_cell_spotcenter_H8[1]
+
+    # check that values are the same
+    if not almost_equal(delta_y1, delta_y2):
+        raise ValueError
+
+    # compare A1 with A8: should have same x
+    delta_x1 = mppc_cell_spotcenter_A1[0] - mppc_cell_spotcenter_A8[0]  # pos: CCW rotation, neg: CW rotation
+    # compare H1 with H8: should have same x
+    delta_x2 = mppc_cell_spotcenter_H1[0] - mppc_cell_spotcenter_H8[0]
+
+    # check that values are the same
+    if not almost_equal(delta_x1, delta_x2):
+        raise ValueError
+
+    # calculate angle for rotation
+    l = 800 * 8  # length in one dimension of the full mppc image of all 8 cells
+    angle = numpy.arcsin(delta_y1/l)  # [rad]
+    mppc_rotation = angle * 360 / (2 * numpy.pi)  # convert to degree
+    # control
+    angle_cont = numpy.arccos(delta_x1/l)  # [rad]
```

Comment:
And I think this should be:
```
delta_y3 = mppc_cell_spotcenter_A1[1] - mppc_cell_spotcenter_A8[1]
angle = numpy.arctan(delta_x1 / delta_y3)  # [rad]
```

Reply:
Is there a reason you choose this one? How about this one:

```suggestion
    delta_x4 = mppc_cell_spotcenter_A8[0] - mppc_cell_spotcenter_H8[0]
    angle_control = numpy.arctan(delta_y2 / delta_x4)  # [rad]
```